src/utils/forwardrate/forwardrate.js

```diff
@@ -0,0 +1,36 @@
+import { retrieveData } from '../network'
+
+/**
+ * @function retrieve
+ * @memberof module:api.ForwardRate
+ * @static
+ * @param {object} params - object of parameters:
+ * @param {number} params.AMId - Asset Manager ID
+ * @param {number} params.assetIds - Asset Id
+ * @param {object} params.query - Object of query params:<br/>
+ * Available keys are:
+ * <li>`string` businessDateStart</li>
+ * <li>`string` businessDateEnd</li>
+ * <li>`string` tenor</li>
+ * @param {Function} [callback] - Called with two arguments (error, result) on completion
+ * @returns {Promise|null} If no callback supplied, returns a Promise
+ */
+
+export function retrieve({ AMId, assetIds, query }, callback) {
+  const params = {
+    AMaaSClass: 'forwardrate',
+    AMId,
+    assetIds,
```

Comment:
Same here you can't pass `assetIds` to `retrieveData` as `retrieveData` doesn't know what to do with it. You have to build it as `resourceId`.